src/main/kotlin/net/leanix/githubagent/GithubAgentApplication.kt

```diff
@@ -1,9 +1,13 @@
 package net.leanix.githubagent
 
 import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.properties.EnableConfigurationProperties
 import org.springframework.boot.runApplication
+import org.springframework.cloud.openfeign.EnableFeignClients
 
 @SpringBootApplication
+@EnableFeignClients(value = ["net.leanix.githubagent.client"])
```

Comment:
I'm not sure if you need add the value for this annotation here("net.leanix.githubagent.client")